raclients/src/frontend/proxy.rs

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use super::lib::{
+    alloc::{
+        string::{String, ToString},
+        vec,
+    },
+    core::{
+        fmt::{self, Display, Formatter},
+        num::TryFromIntError,
+    },
+    format, Vec,
+};
```

Comment:
Please can you take a look at the old version?
I'd like to avoid `use super::lib::alloc...` but just `use crate::lib::fmt` and where we define the `lib` module we include fmt from std or alloc depending on the features.

It's a common pattern used in no_std library, we should follow it.

Reply:
Will do.